cmd/stop.go

```diff
@@ -19,11 +19,16 @@ var StopCmd = &cobra.Command{
 	Use:   "stop",
 	Short: "Stops a running Dapr instance and its associated app",
```

Comment:
Thanks for your contribution!  Please also update the description here and also in docs\reference\dapr-stop.md and README.md so users know this is possible.